src/views/HomePage.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <main>
+    <TheWelcome content="Home Page" />
+  </main>
+</template>
+
+<script lang="ts">
+import TheWelcome from '@components/TheWelcome.vue';
+
+export default {
+  components: { TheWelcome },
+};
+</script>
```

Comment:
이 문법은 Vue2 문법입니다.
Vue3의 setup을 알아보세요. 
https://vuejs.org/api/composition-api-setup

```suggestion
<script lang="ts" setup>
import TheWelcome from '@components/TheWelcome.vue';
</script>
```